e2e/homepage.spec.ts

```diff
@@ -1,11 +1,49 @@
-import { expect, test } from '@playwright/test';
+import {
+	LOGIN_BUTTON,
+	THREE_BACKGROUND_CANVAS,
+	TOKENS_SKELETONS_INITIALIZED
+} from '$lib/constants/test-ids.constant';
+import { testWithII } from '@dfinity/internet-identity-playwright';
+import { expect, test, type Page } from '@playwright/test';
 
 const testUrl = '/';
 
-test('should display not logged in homepage', async ({ page }) => {
-	await page.goto(testUrl);
+const hideHeroAnimation = async (page: Page): Promise<void> => {
+	await page
+		.getByTestId(THREE_BACKGROUND_CANVAS)
+		.evaluate((element) => (element.style.display = 'none'));
+};
 
-	await page.getByTestId('login-button').waitFor();
+test.describe('logged out user', () => {
+	test.beforeEach(async ({ page }) => {
```

Comment:
curiosity for me: is it necessary to add `test.` before the functions? if not, can we remove it, so that it would be "cleaner"?

Reply:
The idea was to extend this `describe` with other tests related to Homepage and therefore to make use of `beforeEach`, but now as I started working on other tests in a parallel PR, I realised that the structure where 1 spec file = 1 test suite (or a few, in case applicable) will work better for us, similar to this repo - https://github.com/dfinity/gix-components/tree/main/e2e. Having said that, we don't need `beforeEach`, I'll remove it 👍

Reply:
no wait, maybe you misunderstood my comment. My point is, instead of `test.describe`, can we just use `describe`? Or do we need the specific `describe` prop of the playwright test suite?
same question for `beforeEach`. It is a genuine doubt I have

Reply:
Ah, I see. Default `describe` and `beforeEach` are declared in `vitest`. And in case of e2e tests, we want to make sure that we use the ones from Playwright. 
P.S.: we actually don't need `describe` here at all because of the same reason described above in this thread. I'll adjust it.

Reply:
This part is now updated. Resolving the thread.